pkg/component/gardener/dashboard/config/login_config.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2024 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Personal preference 😄 I started with one file, but then thought that it might provide a better overview to split into two files, given that this all is in their own `config` package.

Reply:
Fair point. At least `config.go` grew a bit more with additional commits. We can leave it as is from my point of view.